dist/index.html

```diff
@@ -5,7 +5,7 @@
     <title>
       React + Redux - User Registration and Login Example & Tutorial
     </title>
-    <link rel="stylesheet" href="../src/StyleSheet/main.css" />
+    <link rel="stylesheet" href="./main.css" />
     <!--https://netdna.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css-->
```

Comment:
Please remove ghost code (There is a topic in the forum if you are unsure what to do)

Reply:
Thank you. I just did